### PR TITLE
feat: migrate stories of Tooltip, TooltipIcon, Touchable, Typography

### PR DESCRIPTION
**Pull request by DorianMaliszewski**
## Type

- Migration

#### What is expected?

Stories in tsx format

#### The following changes where made:

1. Change stories

2. Export types

3. Improve typing

### CODE REVIEW

**Comment by vincentaudebert** [comment]
Two comments: 

Weird behaviour on default tooltipIcon. Does it need a container maybe?
Screenshot 2021-11-15 at 13 34 38

Should we set a "disabled" mouse icon on disabled touchable elements?
Screenshot 2021-11-15 at 13 35 26

**Review by vincentaudebert — COMMENTED** [review]
Two comments: 

Weird behaviour on default tooltipIcon. Does it need a container maybe?
Screenshot 2021-11-15 at 13 34 38

Should we set a "disabled" mouse icon on disabled touchable elements?
Screenshot 2021-11-15 at 13 35 26

File: src/components/Tooltip/__stories__/index.stories.tsx [hunk 1]
```diff
@@ -127,51 +123,56 @@ import { Box, Button, FlexBox } from '../..'
...
-  With an empty text it will only show children and no tooltip.
-</Description>
+  </>
+)
+
+Children.parameters = {
+  docs: {
+    description: {
+      story:
+        ' Here is a set of examples so you see different type of components having a tooltip.',
```
Comment (QuiiBz) [inline src/components/Tooltip/__stories__/index.stories.tsx:133]:
```suggestion
        'Here is a set of examples so you see different type of components having a tooltip.',
```

**Review by QuiiBz — COMMENTED** [review]
Tooltips variants text are invisible because the background should be black:
Screenshot 2021-11-16 at 10 25 34

**Review by matthprost — APPROVED** [review]

File: src/components/Tooltip/__stories__/index.stories.tsx [hunk 2]
```diff
@@ -127,51 +123,56 @@ import { Box, Button, FlexBox } from '../..'
...
-
-<Description>
-  With an empty text it will only show children and no tooltip.
-</Description>
+  </>
+)
+
+Children.parameters = {
+  docs: {
+    description: {
```
Comment (vincentaudebert) [inline src/components/Tooltip/__stories__/index.stories.tsx:131]: c'est pas story > description ?

File: src/components/Tooltip/__stories__/index.stories.tsx [hunk 3]
```diff
@@ -127,51 +123,56 @@ import { Box, Button, FlexBox } from '../..'
...
+          {placement}
+        </Tooltip>
+      </Box>
+    ))}
+  </FlexBox>
+)
+
+Placement.parameters = {
+  docs: {
+    description: {
```
Comment (vincentaudebert) [inline src/components/Tooltip/__stories__/index.stories.tsx:152]: c'est pas story > description ?

**Review by QuiiBz — APPROVED** [review]

**Pull request merged**